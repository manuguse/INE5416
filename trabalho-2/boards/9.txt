- - - - - - - 2
- 3 - - 5 - 3 -
- - - - - 6 - 4
- 2 - 6 - - 2 -
- - - 4 6 - 4 -
- - 5 - - - - -
2 - - - - - 7 -
- - - - - - - -