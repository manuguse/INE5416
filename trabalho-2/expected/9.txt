2 1 2 1 2 1 6 2
5 3 5 2 5 4 3 1
4 1 4 1 2 6 5 4
3 2 1 6 1 3 2 1
2 3 7 4 6 5 4 6
1 2 5 1 3 4 1 5
2 1 2 3 2 3 7 3
1 3 1 2 1 2 1 2